packages/@n8n/plugin-sdk/tsconfig.backend.json

```diff
@@ -0,0 +1,13 @@
+{
+	"extends": "@n8n/typescript-config/tsconfig.common.json",
+	"compilerOptions": {
+		"tsBuildInfoFile": "./node_modules/.tmp/tsconfig.app.tsbuildinfo",
+		"noEmit": false,
+		"strict": true,
```

Comment:
From a quick check `strict` is already `true` in the parent config. Are all these overrides necessary? Same for the other one.

Reply:
Hmm, that's correct. Removed extra config.